### PR TITLE
CommCare for ICT4D Teams

### PR DESCRIPTION
**Pull request by hemanthReddy06**
- Added new page CommCare for ICT4D Teams
- Copy updates for  CommCare for M&E

### CODE REVIEW

File: dimagi/pages/templates/sections/commcare/commcare_ict4d/tools.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        Your teams have been waiting for a tool that actually aligns with 
+        their needs. Meet the ambitions <br class="hide-until-medium" /> of your Program and M&E teams with 
+        custom-built CommCare applications and powerful data tools.
+        {% endblocktrans %}
+      </p>
+	</div>
+	
+    <div class="off-grid white-lazy-bg"
+       data-lazybg="true">
+    	<div class="wistia-container">
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/commcare_ict4d/tools.html:25]: indentation here needs to be fixed

File: dimagi/pages/templates/sections/commcare/commcare_ict4d/tools.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        {% endblocktrans %}
+      </h2>
+      <p class="lead-text">
+        {% blocktrans %}
+        Your teams have been waiting for a tool that actually aligns with 
+        their needs. Meet the ambitions <br class="hide-until-medium" /> of your Program and M&E teams with 
+        custom-built CommCare applications and powerful data tools.
+        {% endblocktrans %}
+      </p>
+	</div>
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/commcare_ict4d/tools.html:21]: indentation here needs to be fixed, not clear on first glance what this div belongs to

File: dimagi/pages/templates/sections/commcare/commcare_ict4d/track_program.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load dimagidotcom %}
+
+<section class="align-center blue-theme accent-gray-bg border-top" id="track_program">
+  <div class="content pad-vert-medium">
+    <h2 class="content-width-medium normalcase pad-bottom-small theme-color">
+        {% blocktrans %}
```
Comment (biyeun) [inline dimagi/pages/templates/sections/commcare/commcare_ict4d/track_program.html:7]: indentation looks off

**Review by biyeun — CHANGES_REQUESTED** [review]
just a couple code best practices requests with indentation changes but otherwise looking good. happy to merge when those are done

**Comment by hemanthReddy06** [comment]
@biyeun Updated with indentation changes.

**Review by biyeun — APPROVED** [review]

**Pull request merged**